internal/table/table.go

```diff
@@ -103,6 +104,11 @@ func (t *Table) calculateWidth(maxTableWidth int) ([]int, int) {
 		colWidthsCombined += colWidths[n]
 	}
 
+	var verticalHeaderWidth int
+	if len(colWidths) > 0 && t.Vertical {
+		verticalHeaderWidth = colWidths[0]
+	}
```

Comment:
This is only used on `rescaleColumns` and is based on inputs that are also passed to that function. So I suggest moving this logic inside `rescaleColumns`.

But also; what is the rationale here? Comment explaining this would be good.

Reply:
Added a comment, but left this here. There is a call further down to an `equalizeWidths` function that changes the column widths. Before that call the value of the first column is the original header width, which we want to respect when rescaling. I think it makes sense to preserve this, equalize all of the columns, and then ensure the header width is respected when rescaling, but I'm open to move this somewhere else or take a different approach.

Reply:
Ok. I think you missed the first part of my comment though:

> This is only used on rescaleColumns and is based on inputs that are also passed to that function. So I suggest moving this logic inside rescaleColumns.

Reply:
Ah sorry, I didn't explain well exactly what's happening here. 

The call to `equalizeWidths` changes the actual values in the `colWidths` slice. So we have to grab the column width of the first entry in the slice at this point as this is the max width of the headers column. So `verticalHeaderWidth` can be a different value from `colWidths[0]` after the `equalizeWidths` call.